apps/bwclk/app.js

```diff
@@ -100,21 +100,63 @@ let imgLock = function() {
  * Clock Info
  */
 let clockInfoItems = clock_info.load();
+
+// Add some custom clock-infos
+function weekOfYear() {
```

Comment:
Needs change to `let weekOfYear = function() {` to not give ram leak.

Reply:
Oh wow I was not aware of that! Its fixed now, thanks a lot!

Reply:
Feel free to unresolve in case something is still wrong :)